File: tests/conformance/fixtures/typescript/25-evidence-required.ts
import { Pipeline } from '../../../../sdk/typescript/src/index';

const p = new Pipeline();
p.task('test')
  .run('go test ./...')
  .taskType('test')
  .evidenceRequired([
    {
      type: 'file',
      name: 'coverage',
      required: true,
      visibility: 'local',
      predicate: 'non_empty',
      ref_pattern: 'coverage.out',
    },
  ]);
p.emit();

File: tests/conformance/fixtures/rust/25-evidence-required.rs
use sykli::{EvidenceRequirement, Pipeline, TaskType};

fn main() {
    let mut p = Pipeline::new();
    p.task("test")
        .run("go test ./...")
        .task_type(TaskType::Test)
        .evidence_required(&[EvidenceRequirement::file_non_empty("coverage", "coverage.out")]);
    p.emit();
}
